### ISSUE TITLE
Results about the CrowdPose dataset

### ISSUE THREAD

**Issue by zijinhaoyue** [open-mmlab/mmpose#308]
Thanks for the great repos. I retrain the model of pose_resnet_50_256x192 config on my own machine, and i got the 63.1 mAP results based on test dataset with provided detection bboxes. However, when i use the ground truth bboxes during inference, i got 70.8 mAP on test set and 55.3 on val set, it is really confused with the great differences, but i cannot find the reason, is there any changes between val and test? In addition, the annotation files are seems different with the crowd pose dataset, is there any difference?

**Comment by jin-s13** [comment]
63.1 mAP is normal (similar to our released pre-trained model 63.4mAP).

In our experimental settings, we follow [HigherHRNet](https://arxiv.org/abs/1908.10357) to train on train+val datasets and evaluate on test datasets. So I am confused about `55.3 on val set`. What is the val set in your case?

For annotation differecne:
In the original crowdpose dataset [annotation](https://github.com/Jeff-sjtu/CrowdPose), the image `width` and `height` were reversed, and we fixed this bugs.

**Comment by zijinhaoyue** [comment]
Thanks for your reply.
I just train the model on the train dataset, so i evaluate the model both on test set and val set, the val set is the provided annotations correspondenced with crowdpose_val.json.

**Comment by zijinhaoyue** [comment]
The problem may be caused by the 'width' and 'height' on the val set, the image width and height are right on val set initial annotation file, the results turn to be normal (69.8 mAP) when i use the initial val annotation file in crowd pose dataset. 
Thanks for the discuss!

**Comment by jin-s13** [comment]
Thanks for reporting. 
We will update the annotation files in mmpose accordingly.
